README.md

```diff
@@ -1,2 +1,100 @@
 # Rest7Contract
-Repository to discuss the new REST API contract for DSpace 7 https://wiki.duraspace.org/display/DSPACE/DSpace+7+UI+Working+Group
+Repository to discuss the new REST API contract for DSpace 7 <https://wiki.duraspace.org/display/DSPACE/DSpace+7+UI+Working+Group>
+
+## Use of the HTTP Verbs and HTTP Response CODE
+
+### On collection of resources endpoints
+- POST
+Adds a new element to the collection.
+
+- GET
+Returns the first page of the resources in the collection
+
+### On single resource endpoints
+- GET
+Returns a single entity.
+
+- HEAD
+Returns whether the item resource is available.
+
+- PUT
+Replaces the state of the target resource with the supplied request body.
+
+- PATCH
```

Comment:
We can use PATCH to update the value of a metadata without touch (and send) the information about the metadatafield, place, authority, etc.